web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -124,6 +125,14 @@
           </div>
         {/if}
 
+        {#if asset.type === AssetTypeEnum.Image && asset?.exifInfo?.projectionType === ProjectionTypeEnum.Equirectangular}
```

Comment:
No need to use an optional chaining for the asset: `asset?.exifInfo?.projectionType` -> `asset.exifInifo?.projectionType`

Reply:
fixed